octavia-cli/octavia_cli/list/commands.py

```diff
@@ -0,0 +1,51 @@
+#
+# Copyright (c) 2021 Airbyte, Inc., all rights reserved.
+#
+
+from typing import List
+
+import click
+
+from .connectors_definitions import (
+    DestinationConnectorsDefinitions,
+    SourceConnectorsDefinitions,
+)
+
+
+@click.group("list", help="List existing Airbyte resources.")
+@click.pass_context
+def _list(ctx: click.Context):  # pragma: no cover
+    pass
+
+
+@click.group("connectors", help="Latest information on supported sources and destinations connectors.")
+@click.pass_context
+def connectors(ctx: click.Context):  # pragma: no cover
+    pass
+
+
+@connectors.command(help="Latest information on supported sources.")
+@click.pass_context
+def sources(ctx: click.Context):
```

Comment:
@alafanechere it looks like these commands are currently implemented to list source/destination connector **definitions**, i.e. list all of the connector docker images that the user can use to actually create a "Source" or a "Destination". 

I think the naming here might be a bit misleading, as the command is `octavia list connectors sources` rather than something like `octavia list connectors source_definitions`. It may also be useful to have a true `list sources` command that returns the Sources that have actually been created in the workspace, i.e. the result of this API endpoint: https://airbyte-public-api-docs.s3.us-east-2.amazonaws.com/rapidoc-api-docs.html#post-/v1/sources/list. And similarly for Destinations.

What do you think?

Reply:
Thank you for the explanation! That makes a lot more sense to me now - I think the naming of those commands looks good, and I agree that the word `connector` is more meaningful to the user than `definition`.